generators/client/templates/react/webpack/webpack.dev.js.ejs

```diff
@@ -83,9 +84,6 @@ module.exports = async (options) => webpackMerge(await commonConfig({ env: ENV }
       target: 'ws://127.0.0.1:<%= serverPort %>',
       ws: true
     }<% } %>],
-    watchOptions: {
```

Comment:
Are node modules automatically ignored now?

Reply:
it seems https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md